<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222433;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>791</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="QT_camera">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Câmera (1)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="QT_webcam">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Webcam</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="QT_video">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
         </property>
         <property name="text">
          <string>Vídeo do Computador</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="QT_listaVideos">
         <property name="styleSheet">
          <string notr="true">QComboBox{
	color: rgb(255, 255, 255);
	background-color: rgb(25, 25, 33);
	border-radius:0px;
}
QComboBox:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(25, 25, 33);
	border-radius:0px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
